scripts/circle/run_heartbeat_demo_docker-circle.sh

```diff
@@ -8,8 +8,8 @@ echo "Starting Up Heartbeat Demo Test..."
 
 # run some ursulas
 docker-compose up -d
-echo "Wait for Ursula learning to occur"
-sleep 30
+
+echo "wait a few seconds for initting and learning"
```

Comment:
Is this message needed since we are no longer doing a `sleep`? as opposed to 

```suggestion
echo "running heartbeat demo"
```

Reply:
👍